bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html

```diff
@@ -99,21 +96,13 @@ <h3 id="Level1a">{{ _('Level 1a - Named Voucher') }}</h3>
 <table class="table module-table">
   <tr>
     <td>mozilla.org/trunk</td>
-    <td>David Boswell or Reed Loden</td>
+    <td>Reed Loden</td>
   </tr>
   <tr>
     <td>mozilla.com/tags/production</td>
     <td>John Slater</td>
```

Comment:
I know John has gone but I didn't know what to replace it with. If you are trying to stop using these two repos (note there are two with his name on) then we can just remove the entries. Does that work for you guys?